recursion/core-v2/src/lib.rs

```diff
@@ -83,3 +85,20 @@ pub enum MemAccessKind {
     Read,
     Write,
 }
+
+// -------------------------------------------------------------------------------------------------
```

Comment:
nit: weird line

Reply:
is it the empty line or the `// ------` comment? The comment is just following Tej's formatting in the rest of the file.